rpcs3/Emu/RSX/GCM.cpp

```diff
@@ -762,6 +762,34 @@ rsx::primitive_type rsx::to_primitive_type(u8 in)
 	throw new EXCEPTION("Unknow primitive type %d", in);
 }
 
+enum
+{
+	CELL_GCM_WINDOW_ORIGIN_TOP = 0,
```

Comment:
Better, but why move them from the header? I don't think there's any reason to split up the CELL_GCM enum.